src/Utilities/MakeNaN.hpp

```diff
@@ -0,0 +1,40 @@
+// Distributed under the MIT License.
```

Comment:
The header name I think would be clearest if it were `MakeSignalingNan.hpp`. I don't think we'll ever have interest in a non-signaling NaN, and if we do we can rename it then.

Reply:
Could you please also ultimately add this utility in a separate commit before the Variables stuff?